CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 ## 3.21.0 (2023-19-01)
 
 - feat: Add deletion of stack after destroy (remove flag)
+- feat: Add support for an `install` command, similar to
+  [setup-pulumi](https://github.com/marketplace/actions/setup-pulumi)
+
+--
```

Comment:
What's with this trailing `--`?

Reply:
Not sure why this was added. Will fix.
(Somehow I didn't look over the changelog thoroughly.)

Reply:
Figured out where the `--` came from.
The `--` separates the unreleased section from the rest. (https://github.com/pulumi/actions/blob/master/CONTRIBUTING.md)
However, because the original changelog entry was added below a then-unreleased entry,
when git merged these changes with master, it placed this entry under the original unreleased entry.